server/server/models/user.py

```diff
@@ -1,43 +1,24 @@
 """SQLAlchemy model for table users"""
 
 import json
-import decimal, datetime
+import decimal
+import datetime
+import transaction
 
 from passlib.hash import pbkdf2_sha256
 from sqlalchemy import Column, Integer, String, DateTime, ForeignKey, bindparam
 from sqlalchemy.orm import relationship
```

Comment:
You have three lines like `from sqlalchemy.orm import ...`
I think better to call all imports from one module in one line